interfaces/smtp/v0/README.md

```diff
@@ -0,0 +1,64 @@
+# `smtp`
+
+## Overview
+
+This relation interface describes the expected behavior between of any charm claiming to be able to interface with a SMTP Provider and the SMTP Provider itself.
+
+## Usage
+
+In most cases, this will be accomplished using the [smtp library](https://github.com/canonical/smtp-integrator-operator/blob/main/lib/charms/smtp_integrator/v0/smtp.py), although charm developers are free to provide alternative libraries as long as they fulfil the behavioural and schematic requirements described in this document.
+
+## Direction
+
+The `smtp` interface implements a provider/requirer pattern.
+The requirer is a charm that requires SMTP details to connect to an SMTP server, and the provider is a charm holding those details.
+
+```mermaid
+flowchart TD
+    Provider -- host, port, user, password_id, auth_type, transport_security, domain --> Requirer
+```
+
+or alternatively, if secrets are not supported by both sides
+
+```mermaid
+flowchart TD
+    Provider -- host, port, user, password, auth_type, transport_security, domain --> Requirer
+```
+
+## Behavior
+
+The requirer and the provider must adhere to a certain set of criteria to be considered compatible with the interface.
+
+### Provider
+
+- Is expected to provide the host in the relation databag.
```

Comment:
I usually try to use "publish" in this context to avoid confusion with the provider/requirer terminology, but that's a personal preference
```suggestion
- Is expected to publish the `host` in the relation databag.
```

Reply:
Good point. Thx